packages/babel-preset-env/test/fixtures/bugfixes-babel-8/edge-default-params-chrome-70/stdout.txt

```diff
@@ -19,14 +20,8 @@ Using plugins:
   transform-logical-assignment-operators { chrome < 85 }
   transform-nullish-coalescing-operator { chrome < 80 }
   transform-optional-chaining { chrome < 80 }
-  syntax-json-strings
-  syntax-optional-catch-binding
-  syntax-async-generators
-  syntax-object-rest-spread
   transform-export-namespace-from { chrome < 72 }
   transform-modules-commonjs
   transform-dynamic-import
-  syntax-top-level-await
-  syntax-import-meta
```

Comment:
These changes are unexpected but desired: The Babel 8 debug log should not include those legacy syntax plugins. It seems to me that somehow the `validateLogs` option accepts the actual log to be a subset of the expected log, and eventually they are updated in this PR because the ERM transform is added.